docker/nginx.conf

```diff
@@ -0,0 +1,48 @@
+worker_processes      1;
+
+events {
+  worker_connections  1024;
+}
+
+http {
+  include             mime.types;
+  default_type        application/octet-stream;
+
+  sendfile on;
+
+  keepalive_timeout   65;
+
+  gzip on;
+  gzip_static on;
+  gzip_disable "msie6";
+
+  gzip_vary on;
+  gzip_types text/plain text/css application/javascript;
+
+  map $request_method $access_control_max_age {
+    OPTIONS 1728000; # 20 days
+  }
```

Comment:
I don't know the implications of this, but brought it in because it was expected by other parts of the "configurator" code brought in from swagger-ui

Reply:
*Part of the cors.conf file, not the configurator js files